framework/aster-frame/src/arch/x86/mod.rs

```diff
@@ -99,3 +99,59 @@ fn enable_common_cpu_features() {
         });
     }
 }
+
+/// Copies `count * size_of::<T>()` bytes from `src` to `dst`.
+/// The source and destination may overlap.
+///
+/// If the source and destination will never overlap, `copy_nonoverlapping` can be used instead.
+///
```

Comment:
Add a Performance section to explain why these methods are provided even though their functionality is equivalent to their counterparts provided by `core::ptr`.

Reply:
OK, I will add it